DependencyInjection/EventBandExtension.php

```diff
@@ -67,15 +69,30 @@ private function loadAmqpTransport(array $config, ContainerBuilder $container)
             return $camelized;
         };
 
+        $camelizeKeyRecursive = function (array $config) use (&$camelizeKey) {
```

Comment:
Why not just only have one recursive camelizeKey function? It seems a bit redundant to have them both.